kyuubi-server/src/main/scala/org/apache/kyuubi/operation/KyuubiOperationManager.scala

```diff
@@ -142,6 +142,14 @@ class KyuubiOperationManager private (name: String) extends OperationManager(nam
     addOperation(operation)
   }
 
+  override def getQueryId(operation: Operation): String = {
+    val kyuubiOperation = operation.asInstanceOf[KyuubiOperation]
+    val client = kyuubiOperation.client
+    val handle = kyuubiOperation.remoteOpHandle()
+    val queryId = client.getQueryId(handle).getQueryId
```

Comment:
what if remote handle is null ?

Reply:
return null if remote handle is null
I will fix it later, thanks.